addons/mail/static/src/discuss/call/common/call_actions.js

```diff
@@ -0,0 +1,35 @@
+import { registry } from "@web/core/registry";
+
+export const callActionsRegistry = registry.category("discuss.call/actions");
+
+callActionsRegistry
+    .add("mute", {
+        isActive(session) {
+            return session?.isSelfMuted;
+        },
+        icon: "fa-microphone-slash",
```

Comment:
Note that when you don't need `this`, you can pass an arrow function, so 3 LOCs becomes 1:
```js
llActionsRegistry
    .add("mute", {
        isActive: (session) => session?.isSelfMuted,
        icon: "fa-microphone-slash",
    })
```

This doesn't look much, but the less we have to scroll the better. Some similarities with keeping content inline as much as we can in XML: macro-view of file content has value :)